virttest/qemu_vm.py

```diff
@@ -661,6 +661,9 @@ def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                     dev.set_param('mq', mq)
                 if vectors:
                     dev.set_param('vectors', vectors)
+                if failover:
+                    failover = 'on'
+                    dev.set_param('failover', failover)
```

Comment:
Shouldn't `failover` in L666 be the one that is passed into the function in L612?

Reply:
I have updated. the test is passed. 

    -device vfio-pci,host=3b:10.1,id=id_3b.10.1,failover_pair_id=net0,bus=pcie-root-port-3,addr=0x0 \
    -device pcie-root-port,id=pcie-root-port-4,port=0x4,addr=0x1.0x4,bus=pcie.0,chassis=5 \
    -device virtio-net-pci,mac=22:2b:62:bb:a9:82,id=net0,failover=on,netdev=idRyvwdm,bus=pcie-root-port-4,addr=0x0  \
    -netdev tap,id=idRyvwdm,fd=18  \

Reply:
One question, does `failover_pair_id` relate to `failover=on`? I mean, can we set `failover_pair_id` with `failover=off`?

Reply:
failover_pair_id need equal to nic's device_id, not related failover=on or off,